File: 프로그래머스/unrated/181894. 2의 영역/2의 영역.py
def solution(arr):
    arr_index = []
    answer = []
    for i in range(len(arr)):
        if arr[i]==2:
            arr_index.append(i)
    if arr_index:
        min_index = min(arr_index)
        max_index = max(arr_index)
    else:
        return [-1]
    if min_index == max_index:
        answer.append(arr[max_index])
    else:
        for i in range(min_index,max_index+1):
            answer.append(arr[i])
    return answer

File: 프로그래머스/unrated/181894. 2의 영역/2의 영역.js
function solution(arr) {
    let result = [];
    if(arr.includes(2)){
        let a = arr.indexOf(2);
        let b = arr.lastIndexOf(2);
        for(let i=a;i<=b;i++){
            result.push(arr[i]);
        }
        return result;
    }else{
        return [-1];
    }
}